simulator/learning_based_vehicle_model/include/learning_based_vehicle_model/model_connections_helpers.hpp

```diff
@@ -19,9 +19,10 @@
 #include <vector>
 
 std::vector<double> fillVectorUsingMap(
-  std::vector<double> vector1, std::vector<double> vector2, std::vector<int> map, bool inverse);
+  std::vector<double> vector1, std::vector<double> vector2, const std::vector<int> & map,
```

Comment:
The current usage is true, so the operation is to write vector2 to vector1.
Therefore, I don't think there is any problem in using it as a reference.
However, if the first true in the code below were false, model_signals_vec would be rewritten to py_inputs, and the second operation in fillVectorUsingMap would be unintended.

Perhaps such usage is not intended, but looking at the function alone, I think it could be used in such a way.
So I personally think it would be better to leave the value passing as is, but I would like to get any opinions on this.
```
  py_inputs =
    fillVectorUsingMap(py_inputs, model_signals_vec, map_sig_vec_to_py_model_inputs, true);
  py_state = fillVectorUsingMap(py_state, model_signals_vec, map_py_model_outputs_to_sig_vec, true);
```

Reply:
@kobayu858 
> So I personally think it would be better to leave the value passing as is, but I would like to get any opinions on this.

Agreed to your opinion!!